src/lib/middleware/redisPageCache.ts

```diff
@@ -55,6 +56,12 @@ export function redisPageCacheMiddleware(
     return next()
   }
 
+  // TODO: remove once "artist_grid_manual_curation_trial" A/B test is over
+  // Skip targeted artist routes
+  if (TARGETED_ARTIST_ROUTES.includes(req._parsedUrl.pathname)) {
+    return next()
+  }
```

Comment:
**praise**: I just QA'd the review app again and I can no longer reproduce the mismatch so this seems to have worked! 🙌🏼